vars/onWorker.groovy

```diff
@@ -3,11 +3,19 @@
 //import vars.withTimeout
 //import vars.withVirtualenv
 
-// How many test-workers we run in parallel, by default.
-// This no longer needs to be consistent across jobs, but
-// most jobs using the ka-test-ec2 workers use it by default.
+
+// NOTE: We currently use the following workers in e2e-related jobs in parallel:
+// a) 20 workers for the `e2e-test` job.
+// b) 1 worker for the `e2e-test-cypress` job.
+// TODO(FEI-4888): Modify this value once we turn off the Python e2e tests. We
+// could probably reduce this to the number of workers we use in the
+// `webapp-test` job.
+
+// How many test-workers we run in parallel, by default. This no longer needs to
+// be consistent across jobs, but most jobs using the ka-test-ec2 workers use it
+// by default.
 def defaultNumTestWorkerMachines() {
-   return 20;
+   return 21;
```

Comment:
**question:** That makes me think if there's another way to modify the `firstinqueue-priming` job to use a separate primed instance/machine. I'm not fully sure how Jenkins labels and jobs IDs play in this context:

https://github.com/Khan/jenkins-jobs/blob/2a003cd76d1fb59685e0d34adf795fcc8c8cf266/jobs/firstinqueue-priming.groovy#L109-L130

cc @csilvers in case you have any thoughts on this.

Reply:
Yeah, I think the right way to do this is to keep defaultNumTestWorkerMachines and change firstinqueue-priming.groovy to something like:
```
).addStringParam(
   "NUM_WORKER_MACHINES",
   """How many worker machines to use.  Note one worker machine is
   reserved for cypress end-to-end test running.""",
   (onWorker.defaultNumTestWorkerMachines() + 1).toString()
```
You may want a more descriptive doc-change.